_config.yml

```diff
@@ -0,0 +1,60 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+#
+# If you need help with YAML syntax, here are some quick references for you: 
+# https://learn-the-web.algonquindesign.ca/topics/markdown-yaml-cheat-sheet/#yaml
+# https://learnxinyminutes.com/docs/yaml/
+#
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: Latex-access
+name: Latex-access Contributors
+tagline: "A website dedicated to LaTeX to Braille translation."
```

Comment:
I meant to duplicate the description in index.md with what was in the config YAML file but forgot about it. The description in index.md would have overridden the site global descriptor tag anyway. I have now left the long description in index.md and included a tagline in place of the long description in _config.yml.

According to the [Jekyll-seo-tag](https://github.com/jekyll/jekyll-seo-tag) [basic usage documentation](https://github.com/jekyll/jekyll-seo-tag/blob/master/docs/usage.md), this is a long description but a shorter one can be provided in frontmatter with the &ldquo;tagline&rdquo; tag. The keywords meta HTML tag does not appear to be supported in the relevant Jekyll plugin; however, I do not think this matters (Google my personal site as a case in point). Most of the key words are contained in the title and description, both of which are utilised by Jekyll-seo-tag and I wonder if this could be the reason why the developers don't seem too worried about it. I think you'd be pretty safe to merge this as it is probably reading too much into (and relying too much on) one meta tag to the exclusion of many others. If you are still concerned about it then you could build the site locally and see what happens but as stated, it should be fine in all likelihood.